README.md

```diff
@@ -77,19 +77,24 @@ The diagram below describes how the application accesses the configuration.
 
 ## Testing
 
-The tests rely on the test environment set up by controller-runtime.  This
-enables the operator to get initialized against a control pane just like it
-would against a real Openshift cluster.
+The tests rely on the test environment set up by controller-runtime.  This enables the operator to 
+get initialized against a control pane just like it would against a real Openshift cluster.
+
+1. Download and install `kubebuilder`.
+[See install instructions here](https://book.kubebuilder.io/quick-start.html#installation).
+2. Make sure that `kubebuilder` is present in your `PATH`. It is recommended you append it to your
+`PATH` in your `.bashrc`, `.zshrc`, or similar.
+3. Download and install `kustomize`.
```

Comment:
Kustomize isn't required for the current tests - is this required for the new unit tests you added?

Reply:
I was getting some errors about kustomize not being in my PATH when I ran the tests initially :man_shrugging:

Reply:
I think you need kustomize to do the initial code generation done by the SDK.  Worst case devs install kustomize when they don't have to, which doesn't seem that bad.